types/convict/convict-tests.ts

```diff
@@ -1,6 +1,6 @@
 import * as convict from 'convict';
-import * as validator from 'validator';
-import { safeLoad } from 'js-yaml';
```

Comment:
Same as above here as it must be a function

Reply:
I thought about doing that, but it would be a significant change to this file, and it looks like the file itself was copy-pasted from the original package’s documentation, so I wanted to keep this file intact as much as possible. Wdyt?

Reply:
If a new maintainer is joining the crew, it may be very difficult to understand it... I personnaly prefer to keep the stuff easy to understand, maybe the advice from @esetnik or @zoxon can be useful?

Reply:
This is taking longer than I hoped, so instead, I’ll just revert those changes. Wdyt?

Reply:
ok for me